persons.txt

```diff
@@ -0,0 +1 @@
+[{"name":"Andres","height":"88","hair":"black"},{"name":"Andrew","height":"88","hair":"black"},{"name":"Marta","height":"88","hair":"black"}]
```

Comment:
Should you persist this file or it's something only important for the runtime?

Reply:
I added a check if the file exist or is empty, so its not necessary but welcome to have